scalableminds/webknossos
# Add agglomerate skeleton to context menu

### URL of deployed dev instance (used for testing):

### Steps to test:
1. open [dataset](https://www.notion.so/scalableminds/Test-Data-c0563be9c4a4499dae4e16d9b2497cfb#4f27c86bc8fc485e9d04b0dc1b1488c9) with ID mapping 
2. right click context menu on cell and check tooltip on Create Agglomerate Skeleton (should be disabled)
3. activate ID mapping and try Create Agglomerate Skeleton (should work)

### Issues:
- fixes #5762 

------
- [✓] Ready for review
